samples/subsys/display/lvgl/src/main.c

```diff
@@ -144,6 +155,13 @@ int main(void)
 	lv_task_handler();
 	display_blanking_off(display_dev);
 
+#if DT_NODE_EXISTS(DT_ALIAS(pwm_lcd0))
+	if (pwm_set_dt(&pwm_lcd0, PWM_HZ(250U), PWM_HZ(250U))) {
```

Comment:
you can use `pwm_set_pulse_dt(&pwm_lcd0, pwm_lcd0.period)`, that way it picks the frequency from the tree, not that it matters in this case but that 250Hz should not be here

Reply:
I was really hopping to get a review on that because the 250Hz value was something we want in the sample. Thanks for indicate that, I have checked the pwm_* APIs but forgotten to just simply check for relevant members in the struct !!

Reply:
Done